bazel/repository_locations.bzl

```diff
@@ -192,12 +192,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Perfetto",
         project_desc = "Perfetto Tracing SDK",
         project_url = "https://perfetto.dev/",
-        version = "31.0",
-        sha256 = "544c68293590f53391ea4267d5a9b1a4594e1c8216fc5f5ce9d0f1227797922e",
+        version = "32.1",
+        sha256 = "0d1088b4758b3d5f3813178c6de22386329d42407d23aa1479f20dce96e49d78",
         strip_prefix = "perfetto-{version}/sdk",
         urls = ["https://github.com/google/perfetto/archive/v{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2022-11-09",
+        release_date = "2022-02-01",
```

Comment:
I think this needs to be today's date, 2023-02-08

Reply:
The UTC release data on Github for this version is `2023-02-01`

Reply:
Cheers!